python-lib/sharepoint_lists.py

```diff
@@ -64,8 +64,8 @@ def _has_value(response):
 
 
 def assert_list_title(list_title):
-    if not list_title.isalnum():
-        raise ValueError("The list title contains non alphanumerical characters")
+    if "?" in list_title:
```

Comment:
Can you detail the transition between `if not list_title.isalnum():` and `if not list_title.isalnum():`?